ios/engine/KMEI/KeymanEngine/Classes/KeyboardRepository/APILexicalModelRepository.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  APILexicalModelRepository.swift
+//  KeymanEngine
+//
+//  Created by Randy Boring on 3/19/19.
+//  Copyright © 2019 SIL International. All rights reserved.
+//
+
+import Foundation
+import UIKit // for UIDevice
+
+public enum APILexicalModelFetchError: Error {
+  case networkError(Error)
+  case noData
+  case parsingError(Error)
+}
+
+public class APILexicalModelRepository: LexicalModelRepository {
+  private let modelsAPIURL = URLComponents(string: "https://api.keyman.com/model")! //?q=bcp47:en
```

Comment:
The trailing comment can be cleaned up when you're done.

Reply:
Cleaned.